src/app/query/bitcoin/address/address.hooks.ts

```diff
@@ -0,0 +1,21 @@
+// @ts-nocheck
+import { sumNumbers } from '@app/common/utils';
+import { BTC_DECIMALS } from '@shared/constants';
+
+import { useGetTransactionsByAddressQuery } from './transactions-by-address.query';
+import { useGetUtxosByAddressQuery } from './utxos-by-address.query';
+
+// TODO: Export when needed and remove ts-nocheck
+function useGetTransactionsByAddress(address: string) {
+  return useGetTransactionsByAddressQuery(address).data;
+}
+
+function useGetUtxosByAddress(address: string) {
+  return useGetUtxosByAddressQuery(address).data;
+}
+
+function useGetBalanceByAddress(address: string) {
+  const utxos = useGetUtxosByAddressQuery(address).data;
+  if (!utxos) return;
+  return sumNumbers(utxos.map(utxo => utxo.value)).shiftedBy(-BTC_DECIMALS);
+}
```

Comment:
Not necessarily to address here, but I wonder if we should stop creating wrapper hooks that only export the `data` prop (I know we use elsewhere).

Doesn't really do much, adds more code, prevents use of loading states (if needed), and can destructure and rename easy:

```ts
const { data: txs } = useGetTransactionsByAddressQuery(address);
```

Reply:
Yeah, I think this is a good point for sure.